pilopt/src/lib.rs

```diff
@@ -39,6 +39,7 @@ fn remove_constant_fixed_columns<T: FieldElement>(pil_file: &mut Analyzed<T>) {
     let constant_polys = pil_file
         .constant_polys_in_source_order()
         .iter()
+        .filter(|(p, _)| !p.is_array())
```

Comment:
Can you make an issue so someone eventually checks how to propagate this optimization to arrays?

Reply:
-> https://github.com/powdr-labs/powdr/issues/1028